### DISCUSSION TITLE
[Feature Request] Option to Modify Umami Text At The Top Left

### DISCUSSION THREAD

**Discussion by mariushosting** [umami-software/umami#1198] [Ideas]
An option to change the Umami text next to the umami logo in the upper left corner with a custom one such as "Frank Stats" would be very useful and would help personalize the page.

**Comment by mariushosting** [comment]
I don't want to fork it. Let the original great Mike author provide the best settings for us!

**Comment by MagicLike** [comment]
@mariushosting
What about also changing the logo in the upper left corner for e.g. branding like in Nextcloud?

**Comment by mariushosting** [comment]
> @mariushosting What about also changing the logo in the upper left corner for e.g. branding like in Nextcloud?

Yes, also uptime kuma do this.

**Comment by dtonon** [comment]
+1
The Umami brand currently is really prominent, I would like to personalize that spot with a custom text; the Umami credits could be moved to the footer, as usual.

**Comment by julianwitzel** [comment]
Branding or Whitelabel options would be super useful for us. Maybe there could also be an option to add custom CSS to override the defaults from within the settings so that the changes persist in case of an update.
